examples/omnivore/omnivore_inference_demo.ipynb

```diff
@@ -28,8 +28,12 @@
     "from torchvision.io import read_video, _read_video_from_file\n",
     "import torchvision.transforms as T\n",
     "\n",
+    "import sys\n",
+    "# We add the folder: \"<torchmultimodal_repo>/examples\" to path to import the presets\n",
+    "sys.path.append(os.path.dirname(os.path.abspath(\"\")))\n",
+    "\n",
     "import torchmultimodal.models.omnivore as omnivore\n",
-    "from examples.omnivore.data import presets\n",
+    "from omnivore.data import presets\n",
```

Comment:
Follow the import path in https://github.com/facebookresearch/multimodal/pull/379

To do this, we need to add the path to `<torchmultimodal_repo>/examples` to the path.